examples/demo/demo.go

```diff
@@ -642,6 +636,12 @@ func dialogs(ts *core.Tabs) {
 		core.NewText(d).SetType(core.TextSupporting).SetText("A standalone window that opens in the same system window")
 		d.NewWindow().SetNewWindow(false).SetDisplayTitle(true).Run()
 	})
+
+	rw := core.NewButton(wrow).SetText("Resize to content")
+	rw.SetTooltip("Resizes this window to fit the current content")
+	rw.OnClick(func(e events.Event) {
+		wrow.Scene.ResizeToContent(image.Point{0, 40}) // note: size is not correct due to wrapping? #1307
```

Comment:
Again, this is not the place to put the extra and the note.

Reply:
?? where else?

Reply:
image.Pt(0,40)